src/widgets/study-path/ui/study-path.tsx

```diff
@@ -46,7 +46,7 @@ export const StudyPath = ({ path, marked, lang = 'en' }: StudyPathProps) => {
       <section className="study-path container" id="learning-path">
         <div className="study-path content upd">
           <WidgetTitle size="small" mods="asterisk">{title}</WidgetTitle>
-          <Paragraph>{paragraph}</Paragraph>
+          <Paragraph className="study-path-paragraph">{paragraph}</Paragraph>
```

Comment:
```suggestion
          <Paragraph>{paragraph}</Paragraph>
```
As I can see nothing change

Reply:
There are differences in font sizes on media-laptop.

Reply:
[Resolved](https://github.com/rolling-scopes/site/pull/475/commits/c2a047a0dedc9dbacb91d4e2a6adb6117bb32111).